scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -459,6 +462,7 @@ get_transfer()
             if [[ -z "$SOCAT_VERSION" ]]; then
                 wsrep_log_error "******************* FATAL ERROR ********************** "
                 wsrep_log_error "* Cannot determine the socat version. "
+                wsrep_log_error "* Line $LINENO"
```

Comment:
wondering what is the significance of printing LINENO.

Reply:
Makes it easier to locate the error within the code when debugging

Reply:
ok. We can keep it but otherwise error message that is printed anyway helps to trace respective line.